### DISCUSSION TITLE
Welcome to Bed Level Visualizer Discussions!

### DISCUSSION THREAD

**Discussion by jneilliii** [jneilliii/OctoPrint-BedLevelVisualizer#373] [General]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other users of Bed Level Visualizer. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other users.

**Comment by gicalgary** [comment]
Hi,
I installed the plugin to octoprint which is connected to an ender 3 with BLTouch running on SKR mini 2.0. I go to generate the mesh, and it works ok.. except after it generates the mesh it just starts to generate a new mesh immediately.
What can I do to troubleshoot the continuous probing? It's like there's an endless loop.
Thanks

**Reply by jneilliii** [reply]
Never heard that one before. Do you have multiple @BEDLEVELVISUALIZER flags in tir collection gcode?

**Reply by gicalgary** [reply]
Hi.. I'm using the code from the example..
---
G28       ; home all axes
M155 S30  ; reduce temperature reporting rate to reduce output pollution
M190 S65  ; (optional) wait for the bed to get up to temperature
G29 P1    ; automatically populate mesh with all reachable points
G29 P3    ; infer the rest of the mesh values
G29 P3    ; infer the rest of the mesh values again
@BEDLEVELVISUALIZER	; tell the plugin to watch for reported mesh
M420 S1 V ; enabled leveling and report the new mesh
M500      ; save the new mesh to EEPROM
M155 S3   ; reset temperature reporting

**Reply by jneilliii** [reply]
Yeah, looks correct. Try removing all the comments and see if that makes a difference.

**Reply by gicalgary** [reply]
I took out the comments. It completed one pass for the bed level, then started retrieving the mesh immediately

**Reply by jneilliii** [reply]
When you compiled your firmware did you use UBL instead of bilinear? The commands you posted are UBL specific.

**Reply by gicalgary** [reply]
UUUGGGHHHH ! It was so long since I compiled the firmware I forgot what I used 🤦‍♂️ Yes.. bilinear it was

**Reply by jneilliii** [reply]
that's why then, each G29 command was running a probe process. You need to switch your commands to the other from examples.
